Cargo.toml

```diff
@@ -21,6 +21,7 @@ bitcoin = { version = "0.28.1", features = ["use-serde"] }
 log = "^0.4"
 ureq = { version = "~2.2.0", features = ["json"], optional = true }
 reqwest = { version = "0.11", optional = true, default-features = false, features = ["json"] }
+tokio = { version = "1.20.1", features = ["full"] }
```

Comment:
Since tokio is only needed for testing the async version can we make this optional? or if only needed for testing move it to `[dev-dependencies]`.

Reply:
Ah, you're totally right. Moved it to `dev-dependencies` with 8caa69b.